capstone/src/main/resources/static/map.js

```diff
@@ -40,8 +39,28 @@ function createMarker(event,campusMap) {
   const newMarker = new google.maps.Marker({
     map: campusMap,
     title: event.eventTitle,
-    position: eventPosition
+    position: eventPosition,
   });
+  var eventInfoWindow = createInfoWindow(event, campusMap);
+  var eventContent = '<p id=mapContent>'+ event.eventTitle + '</p>';
```

Comment:
How can the user save the event from this view? Would it be possible to provide a link to the event details page? I'm not sure we have a separate event details page though, just a popup.

Reply:
There isn't a dedicated page, it's just a popup. I'm not sure if I could make a link to it or not. Also the events on the map are the saved events from the user